ios/RNTPTDocumentView.m

```diff
@@ -345,6 +345,7 @@ - (void)registerForPDFViewCtrlNotifications:(PTDocumentBaseViewController *)docu
 {
     PTPDFViewCtrl *pdfViewCtrl = documentViewController.pdfViewCtrl;
     PTToolManager *toolManager = documentViewController.toolManager;
+    PTUndoRedoManager *undoManager = toolManager.undoManager;
```

Comment:
Should be the `NSUndoManager` type - the `undoRedoManager` is the `PTUndoRedoManager` (very similar names)
```suggestion
    NSUndoManager *undoManager = toolManager.undoManager;
```